docs/playbook.md

```diff
@@ -42,5 +42,15 @@ Enter NO_PUBKEY here (eg: 6494C6D6997C215E is NO_PUBKEY for above error)
 
 **Solution**
 
-*   sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 6494C6D6997C215E
-*   sudo apt-get update
\ No newline at end of file
+* sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 6494C6D6997C215E
+* sudo apt-get update
+
+### Mount Failed with Error: Current requires cgo or $USER set in environment
```

Comment:
so do you get this error now when panic is removed

Reply:
Yes.